protocols/gossipsub/src/config.rs

```diff
@@ -186,6 +186,10 @@ impl Config {
     /// The time a connection is maintained to a peer without being in the mesh and without
     /// send/receiving a message from. Connections that idle beyond this timeout are disconnected.
     /// Default is 120 seconds.
+    #[deprecated(
+        note = "Set a global idle connection timeout via `SwarmBuilder::idle_connection_timeout` instead."
+    )]
+    #[allow(deprecated)]
```

Comment:
You'll have to put that on the _usage_ site of the method. See the annotations below :)

Reply:
Oops, My bad! I thought was placing it on the deprecated function. 😅
Fixed 8b565ea